NuGet.Config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+    <config>
+        <add key="globalPackagesFolder" value=".nuget" />
```

Comment:
Why do we want to override this?

Reply:
Good catch. The tool I used to generate the package mapping required it. Removed.